developer_menu/menu_view.xml

```diff
@@ -2,7 +2,7 @@
 
 <odoo>
 
-<menuitem id="conf_tech" parent="base.menu_administration" name="🧰" groups="base.group_erp_manager" sequence="100"/>
+<menuitem id="conf_tech" parent="base.menu_administration" name="🧰" groups="base.group_erp_manager" sequence="1"/>
 <menuitem id="model" name="Model" parent="conf_tech" action="base.action_model_model" sequence="10"/>
```

Comment:
There is a new syntax that allow to organize menu hierarchy:
https://github.com/odoo/odoo/blob/16.0/addons/account/views/account_menuitem.xml#L11

Reply:
Je savais pas merci